pcap-int.h

```diff
@@ -113,6 +113,7 @@ struct pcap_opt {
 	int	buffer_size;
 	int	promisc;
 	int	rfmon;		/* monitor mode */
+	int rfmon_selfstart;
```

Comment:
It has been fixed at: https://github.com/the-tcpdump-group/libpcap/pull/512/commits/1c3bf75b09e74efaf63179ef8aba000d2610b415